src/mpcCoreKit.ts

```diff
@@ -784,11 +784,12 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
   }
 
   /**
-   * WARNING: Use with caution. This will export the private key.
+   * WARNING: Use with caution. This will export the private key for secp256k1 KeyType.
    *
    * Exports the private signing key for the current account index.
    */
   public async _UNSAFE_exportTssKey(): Promise<string> {
+    if (!(this.keyType === KeyType.secp256k1)) throw new Error("wrong keytype to call this method");
```

Comment:
We could let this also output the private key scalar for keyType != secp256k1. I think @himanshuchawla009 previously talked about potential use cases for this. I'm undecided, both seems fine.